### DISCUSSION TITLE
release的exe如何修改端口？

### DISCUSSION THREAD

**Discussion by verne91** [1715173329/UnblockNeteaseMusic#333] [Q&A] [answered]

**Comment by pan93412** [comment]
```
node app.js -p HTTP_PORT
node app.js -p HTTP_PORT:HTTPS_PORT
```

如果有解答到您的問題，請幫我點選「Mark as answer」，謝謝！

**Reply by verne91** [reply]
请问exe双击之后就直接弹出一个terminal 显示
INFO: (app) HTTP Server running @ http://0.0.0.0:8080
INFO: (app) HTTPS Server running @ http://0.0.0.0:8081

应该在什么地方输入以上命令？

**Comment by pan93412** [comment, marked as answer]
> 请问exe双击之后就直接弹出一个terminal 显示 INFO: (app) HTTP Server running @ http://0.0.0.0:8080 INFO: (app) HTTPS Server running @ http://0.0.0.0:8081
> 
> 应该在什么地方输入以上命令？

用 cmd 執行？把 exe 拖到 cmd，然後在後面加上 `-p ...`。像這樣：

```bat
> C:\Users\pan93412\Desktop\unm.exe -p [上面提的參數]
```

**Reply by verne91** [reply]
谢谢！
